src/cloudant/design_document.py

```diff
@@ -16,18 +16,18 @@
 API module/class for interacting with a design document in a database.
 """
 from ._2to3 import iteritems_
+from ._common_util import QUERY_LANGUAGE, codify
 from .document import Document
 from .view import View, QueryIndexView
 from .error import CloudantArgumentError, CloudantException
-from ._common_util import QUERY_LANGUAGE
 
 class DesignDocument(Document):
     """
     Encapsulates a specialized version of a
     :class:`~cloudant.document.Document`.  A DesignDocument object is
     instantiated with a reference to a database and
-    provides an API to view management, list and show
-    functions, search indexes, etc.  When instantiating a DesignDocument or
+    provides an API to view management, search index management, list and show
```

Comment:
@ricellis said: "shouldn't this just be index management?
The design document can contain query indexes as well."

Query index management is not handled in `DesignDocument` class. 
Create/delete/etc query index functions are handled in `index` module and the user would call these functions in `CloudantDatabase` (i.e. `CloudantDatabase.create_query_index`)